kafka-rest/src/test/java/io/confluent/kafkarest/integration/ClusterTestHarness.java

```diff
@@ -298,11 +307,23 @@ protected Properties getBrokerProperties(int i) {
   @AfterEach
   public void tearDown() throws Exception {
     log.info("Starting teardown of {}", getClass().getSimpleName());
+    tearDownMethod();
+    log.info("Completed teardown of {}", getClass().getSimpleName());
+  }
+
+  private void tearDownMethod() throws Exception {
+
+    restProperties.clear();
     if (restServer != null) {
       restServer.stop();
       restServer.join();
     }
```

Comment:
This has been replaced with `stopRest()` by some of your previous changes, and indeed the PR is conflicting with `master`. Can you please resolve the conflicts?